src/App.js

```diff
@@ -1,26 +1,171 @@
 import React from 'react';
-import logo from './logo.svg';
 import './App.css';
 
-function App() {
+function Heading() {return <h1 className="Heading">todos</h1>}
```

Comment:
This kind of function is redundant. Better to define needed HTML directly in the component.
It's a good practice to split complex sub-components but not a simple tag with text. It will be hard to maintain further.